js/lightbeam.js

```diff
@@ -113,6 +113,16 @@ async function initLightBeam() {
   websites = await storeChild.getAll();
   renderGraph(websites);
 
+  const saveData = document.getElementById('save-data-button');
+  saveData.addEventListener('click', () => {
+    storeChild.getAll().then( data => {
+      const blob = new Blob([JSON.stringify(data,' ',2)],
```

Comment:
Nit: spaces here between arguments. Again not sure if there is a rule we can turn on.